cdap-ui/app/features/flows/controllers/tabs/runs/runs-detail-ctrl.js

```diff
@@ -16,7 +20,11 @@ angular.module(PKG.name + '.feature.flows')
       event.targetScope.runs.selected = null;
     });
 
-    $scope.selectTab = function(tab) {
+    $scope.selectTab = function(tab, node) {
+      if (tab.title === 'Flowlets') {
```

Comment:
This function should probably be ```goToFlowlet(flowletId)``` right? What we are actually doing is going to the flowlet detailed view but the function name suggests we are just switching the tab. The name kind of mis-leads here.. Thoughts?